releasenotes/notes/oracle-obfuscator-1c25fa49f9f7dd0f.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Switch to the new obfuscator where lots of bugs are fixed.
```

If you don't want to mention the specific bugs, then I'd suggest:

> Switch to the new obfuscator where bugs are fixed.

```suggestion
    Switch to the new obfuscator where bugs such as getting an error when obfuscating `@!` and where comments on DMLs weren't being removed are fixed.
```